libipl/p10/ipl0.C

```diff
@@ -374,28 +387,124 @@ static int ipl_alignment_check(void)
 	return -1;
 }
 
+static int initialize_and_check_clock_chip(uint8_t& clock_count)
+{
+	struct pdbg_target *clock_target;
+	enum pdbg_target_status status;
+	uint8_t data;
+	int rc = 0;
+
+	// initialize output variable.
+	clock_count = 0;
+	
+	ipl_log(IPL_DEBUG, "Istep: soft reset clock, and verify clock status register\n");
+	pdbg_for_each_class_target("oscrefclk", clock_target) {
```

Comment:
Sorry, I missed this. Do we need to quit IPL if one of the oscrefclk initialization is failed?
I think we need to attempt all the clock right?

Reply:
yes, i think it would be better to try on both before exiting. (even though we will fail even if one clock it fails)